src/Provider/CdnAwareProviderInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Provider;
+
+use Sonata\MediaBundle\CDN\CDNInterface;
+use Sonata\MediaBundle\Model\MediaInterface;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+interface CdnAwareProviderInterface
```

Comment:
this is not `pedantic`.

Reply:
I'm sorry, this file is not meant to be added in this PR.
Thank you.